src/SignalR/docs/specs/TransportProtocols.md

```diff
@@ -186,6 +186,32 @@ Long Polling requires that the client poll the server for new messages. Unlike t
 
 A Poll is established by sending an HTTP GET request to `[endpoint-base]` with the following query string parameters
 
+## HTTP Streaming (HTTP/1, HTTP/2, HTTP/3)
+
+HTTP Streaming uses a single bi-directional HTTP request or 2 uni-directional HTTP requests as a single connection (one for input, one for output). If the HTTP client in use supports bi-directional streaming, negotiation can be skipped (if the client knows the server supports the streaming transport), otherwise, it requires a connection already be established using the `POST [endpoint-base]/negotiate` request.
```

Comment:
```suggestion
HTTP streaming uses [chunked transfer coding](https://tools.ietf.org/html/rfc7230#section-4.1) to stream data between the client and the server on a single HTTP request.

If the client supports *simultaneous* bi-directional streaming. then a single HTTP request is used and negotiation can be skipped (if the client has prior knowledge that the server supports the streaming transport). In order to use this transport, the server must be able to send response body chunks while still receiving request body chunks, and the client must be able to receive response body chunks while still sending request body chunks.

For clients that only support response body streaming (such as browsers), negotiation must be performed in order to allocate a connection ID. A single streaming request is used to send data from server to client (as with Server-Sent Events). The HTTP POST pattern described above is used to send data from client to server.
```

Reply:
Updated my suggestion to remove reference to chunking, since that's only how it's done in HTTP/1.1, not HTTP/2+